account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -41,6 +41,7 @@ class AccountPaymentLineCreate(models.TransientModel):
     move_line_ids = fields.Many2many(
         comodel_name="account.move.line", string="Move Lines"
     )
+    move_line_domain = fields.Json()
```

Comment:
Usually this is saved as `Binary`, not `Json`. Has this changed in v17?

Reply:
Well apparently it works fine with both, I'll change it to Binary